packages/istanbul-report-modern-html/src/index.js

```diff
@@ -0,0 +1,119 @@
+            sortKey: 'file',
+            order: 'desc'
+        }
+    );
+    const [summarizerType, setSummarizerType] = React.useState(
+        (routingDefaults && routingDefaults.summarizerType) || 'nested'
+    );
+    const [activeFilters, setFilters] = React.useState(
+        (routingDefaults && routingDefaults.activeFilters) || {
+            low: true,
+            medium: true,
+            high: true
+        }
+    );
+    const childData = React.useMemo(
+        () =>
+            getChildData(
+                sourceData,
+                metricsToShow,
+                activeSort,
+                summarizerType,
+                activeFilters
+            ),
+        [activeSort, summarizerType, activeFilters]
+    );
+    const overallMetrics = sourceData.package.metrics;
+
+    React.useEffect(() => {
+        setLocation(firstMount, activeSort, summarizerType, activeFilters);
+        firstMount = false;
+    }, [activeSort, summarizerType, activeFilters]);
+
+    React.useEffect(() => {
+        window.onpopstate = () => {
+            const routingState = decodeLocation();
+            if (routingState) {
+                setFilters(routingState.activeFilters);
+                setSort(routingState.activeSort);
+                setSummarizerType(routingState.summarizerType);
+            }
+        };
+    }, []);
+
+    return (
+        <>
+            <div className="wrapper">
+                <SummaryHeader
+                    metrics={overallMetrics}
+                    metricsToShow={metricsToShow}
+                />
+                <div className="pad1">
+                    <SummarizerButtons
+                        setSummarizerType={setSummarizerType}
+                        summarizerType={summarizerType}
+                    />
+                    <FilterButtons
+                        activeFilters={activeFilters}
+                        setFilters={setFilters}
+                    />
+                </div>
+                <div className="pad1">
+                    <table className="coverage-summary">
+                        <SummaryTableHeader
+                            onSort={newSort => {
+                                setSort(newSort);
+                            }}
+                            activeSort={activeSort}
+                            metricsToShow={metricsToShow}
+                        />
+                        <tbody>
+                            {childData.map(child => (
+                                <SummaryTableLine
+                                    {...child}
+                                    key={child.file}
+                                    metricsToShow={metricsToShow}
+                                />
+                            ))}
+                        </tbody>
+                    </table>
+                </div>
+                <div className="push" />
+                {/* for sticky footer */}
```

Comment:
comment was in old report. seems invalid?

Reply:
See http://istanbul-modern-report.surge.sh/istanbul-api/lib/input-error.js.html - `Code coverage generated by istanbul at Sun Apr 14 2019 21:32:36 GMT+0200 (Romance Summer Time)` is pinned to the bottom of the page as long as the page is not scrolling.  Unfortunately the same does not happen for the directory view.

Reply:
Fixed